### ISSUE TITLE
Create an event page template

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#65]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

A page template designed for live events, with a built in flight finder.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
